src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotifyLiPClaimantHwFOutcomeHandler.java

```diff
@@ -0,0 +1,117 @@
+    private static final String REFERENCE_TEMPLATE = "hwf-outcome-notification-%s";
+    private final NotificationService notificationService;
+    private final NotificationsProperties notificationsProperties;
+    private final FeatureToggleService toggleService;
+
+    private final Map<String, Callback> callBackMap = Map.of(
+        callbackKey(ABOUT_TO_SUBMIT), this::notifyApplicantForHwFOutcome
+    );
+    private Map<CaseEvent, String> emailTemplates;
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return callBackMap;
+    }
+
+    @Override
+    public String camundaActivityId(CallbackParams callbackParams) {
+        return TASK_ID;
+    }
+
+    private CallbackResponse notifyApplicantForHwFOutcome(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+
+        if (caseData.isLipvLipOneVOne() && toggleService.isLipVLipEnabled()) {
+            sendEmail(caseData);
+        }
+
+        return AboutToStartOrSubmitCallbackResponse.builder()
+            .build();
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    @Override
+    public Map<String, String> addProperties(CaseData caseData) {
+        switch (caseData.getHwFEvent()) {
+            case NO_REMISSION_HWF:
+                return getNoRemissionProperties(caseData);
+            default:
+                return null;
+        }
+    }
+
+    private String getTemplate(CaseEvent hwfEvent) {
+        if (emailTemplates == null) {
+            emailTemplates = ImmutableMap.of(
+                CaseEvent.NO_REMISSION_HWF,
+                notificationsProperties.getNotifyApplicantForHwfNoRemission()
+            );
+        }
+        return emailTemplates.get(hwfEvent);
+    }
+
+    private Map<String, String> getNoRemissionProperties(CaseData caseData) {
+        return Map.of(
+            CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
+            CLAIMANT_NAME, getPartyNameBasedOnType(caseData.getApplicant1()),
+            REASONS, getHwFNoRemissionReason(caseData),
+            TYPE_OF_FEE, caseData.getHwfFeeType().getLabel(),
+            HWF_REFERENCE_NUMBER, caseData.getHwFReferenceNumber(),
+            AMOUNT, caseData.getHwFFeeAmount().toString()
+        );
+    }
+
+    private void sendEmail(CaseData caseData) {
+        if (Objects.nonNull(caseData.getApplicant1Email())) {
+            notificationService.sendMail(
+                caseData.getApplicant1Email(),
+                getTemplate(caseData.getHwFEvent()),
+                addProperties(caseData),
+                String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
+            );
+        }
+    }
+
+    private String getHwFNoRemissionReason(CaseData caseData) {
+        if (caseData.isHWFTypeHearing()) {
+            return caseData.getHearingHwfDetails().getNoRemissionDetailsSummary().getLabel();
```

Comment:
Add null checks for caseData.getHearingHwfDetails() & caseData.getClaimIssuedHwfDetails()

Reply:
I do not consider it necessary. It is a private function inside a handler in which at the time of the call there will always be data in claimIssuedHwfDetails or HearingHwfDetails.